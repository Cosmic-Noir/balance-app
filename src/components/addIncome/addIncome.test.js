import React from "react";
import ReactDom from "react-dom";
import renderer from "react-test-renderer";
import AddIncome from "./addIncome";

describe(`AddIncome Component`, () => {
  it(`Renders without crashing`, () => {
    const div = document.createElement("div");
    ReactDom.render(<AddIncome />, div);
    ReactDom.unmountComponentAtNode(div);
  });

  it(`Renders the UI as expected`, () => {
    const tree = renderer.create(<AddIncome />).toJSON;
    expect(tree).toMatchSnapshot();
  });
});
