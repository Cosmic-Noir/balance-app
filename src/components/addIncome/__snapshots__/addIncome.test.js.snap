// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AddIncome Component Renders the UI as expected 1`] = `[Function]`;
